SSA/Core/Framework.lean

```diff
@@ -140,6 +140,22 @@ Another option, which would be more complicated, but correct by construction, is
 impurely.
 -/
 
+/--
+DialectPrint includes the functions to print the components of a dialect.
+-/
+class DialectPrint (d : Dialect) where
```

Comment:
There is a naming mismatch here. We talk later about the `ToPrint` instances, but call the class `DialectPrint`. Should we just name it `ToPrint`?

Reply:
I renamed this for now to `ToPrint`.